src/v1/json/salutes.json

```diff
@@ -1,24 +1,56 @@
 {
   "salutes": [
+    "Stone.",
     "Rock on!",
+    "We rock!",
     "For Karl!",
+    "Rock solid!",
+    "For Teamwork!",
+    "By the Beard!",
+    "Rock and roll!",
+    "Rock... Solid!",
     "Rock and Stone!",
+    "Rock and Stone.",
```

Comment:
This one can be removed because there is one with an exclamation mark above it and they seem too similar
```suggestion
```

Reply:
This is argueable. I go with the doubled exclamation marks, but this one has a slightly different meaning, don't you think?

Reply:
Yeah I can see that - let's keep both

Rock and Stone!